tools/test.py

```diff
@@ -19,6 +19,10 @@ def parse_args():
     parser.add_argument(
         '--work-dir',
         help='the directory to save the file containing evaluation metrics')
+    parser.add_argument(
+        '--dump',
```

Comment:
Consider to use `--format-only` or some keys that exists in MMDet 2.x for compatibility.

Reply:
We follow the MMCls's args. Do we have any conventions?